ROADMAP.md

```diff
@@ -4,6 +4,9 @@
 - [ ] propagate cost in integrator
 - [ ] restore download and compilation of OOQP
 - [ ] split ocp solve into prepare and feedback
+- [ ] stage transition functions for changing model dimensions
```

Comment:
That would be good to have :) Still have to decide if it is better to make the functions explicit, implicit or both